File: lanjii/lanjii-modules/module-system/system-biz/src/main/java/com/lanjii/sys/entity/SysPost.java
package com.lanjii.sys.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.lanjii.sys.base.AdminBaseMapper;
import com.lanjii.system.api.dto.SysPostDTO;
import com.lanjii.system.api.vo.SysPostVO;
import com.lanjii.framework.mp.base.TenantBaseEntity;
import lombok.Data;
import lombok.EqualsAndHashCode;
import org.mapstruct.Mapper;
import org.mapstruct.factory.Mappers;

/**
 * 系统岗位表(SysPost)表实体类
 *
 * @author lanjii
 */
@Data
@EqualsAndHashCode(callSuper = true)
@TableName("sys_post")
public class SysPost extends TenantBaseEntity {

    /**
     * 岗位ID
     */
    private Long id;

    /**
     * 岗位编码
     */
    private String postCode;

    /**
     * 岗位名称
     */
    private String postName;

    /**
     * 显示顺序
     */
    private Integer sortOrder;

    /**
     * 是否启用（1启用 0禁用）
     */
    private Integer isEnabled;

    /**
     * 备注
     */
    private String remark;

    @Mapper
    public interface SysPostMapper extends AdminBaseMapper<SysPost, SysPostVO, SysPostDTO> {

    }

    public static final SysPostMapper INSTANCE = Mappers.getMapper(SysPostMapper.class);

}

File: lanjii/lanjii-admin-ui/src/types/sys/sysPost.ts
/** 系统岗位 */
export interface SysPost {
  /** 岗位ID */
  id?: number
  /** 岗位名称 */
  postName?: string
  /** 岗位编码 */
  postCode?: string
  /** 显示顺序 */
  sortOrder?: number
  /** 是否启用（1启用 0禁用） */
  isEnabled?: number
  /** 备注 */
  remark?: string
  /** 创建时间 */
  createTime?: string
  /** 更新时间 */
  updateTime?: string
  /** 创建人 */
  createBy?: string
  /** 更新人 */
  updateBy?: string
  /** 删除标志（0-未删除，1-已删除） */
  deleted?: number
}
